package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@guardian/cdk",
   "description": "Generic Guardian flavoured AWS CDK components",
-  "version": "0.0.0",
+  "version": "51.0.0",
```

Comment:
How is this kept up-to-date?

Reply:
Changesets will bump this version in accordance with the changeset (*.md) file created using the Changesets CLI tool.